### PR TITLE
Replace uses of `strings.Split(N)` with `strings.Cut()`

### PR DESCRIPTION
**Pull request by thaJeztah**
Similar to https://github.com/moby/moby/pull/44381

**- Description for the changelog**

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

**Comment by thaJeztah** [comment]
@rumpl ptal 🤗

**Review by vvoland — CHANGES_REQUESTED** [review]
Small typo 😅

File: cli/command/container/opts.go [hunk 1]
```diff
@@ -848,22 +841,22 @@ func parseLoggingOpts(loggingDriver string, loggingOpts []string) (map[string]st
...
 	for key, opt := range securityOpts {
-		con := strings.SplitN(opt, "=", 2)
-		if len(con) == 1 && con[0] != "no-new-privileges" {
-			if strings.Contains(opt, ":") {
-				con = strings.SplitN(opt, ":", 2)
-			} else {
-				return securityOpts, errors.Errorf("Invalid --security-opt: %q", opt)
-			}
+		k, v, ok := strings.Cut(opt, "=")
+		if !ok && k != "no-new-privileged" {
```
[the remark this answers is not in the document]
Comment (thaJeztah) [reply cli/command/container/opts.go:845]: Oh! 💩 🙈 Good catch. I, erm, blame auto-correct, but I likely fat-fingered 😂 .

**Comment by thaJeztah** [comment]
@vvoland fixed the typos 😅 


(I'd love to get rid of some of that logic in the ParseSecurityOpts; I don't think it should all belong in that function, or even in the CLI code)

**Review by vvoland — APPROVED** [review]
LGTM

**Pull request merged**